docs/REST API/rest-api.md

```diff
@@ -4328,11 +4328,11 @@ OS vulnerability data is currently available for Windows and macOS. For other pl
       "software_package": {
         "name": "FalconSensor-6.44.pkg"
         "self_service": false,
-        "last_install": null
       },
       "app_store_app": null    
-      "source": "",
+      "source": "", #ios_apps,ipados_apps
       "status": null,
```

Comment:
Wording the comment this way is a little ambiguous, I think having this and similar comments below be `# | "ios_apps" | "ipados_apps" | ...` would more clearly say "or it could be..."

Reply:
Thanks for the feedback!

Heads up that we're going to pull the comments out before we merge the PR because don't do this for other API endpoints. It would be more work to maintain and would likely fall out of date.